runtime/vm/jvminit.c

```diff
@@ -1823,7 +1823,8 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				IDATA argIndexIgnoreUnrecognizedOptionsEnable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSENABLE, NULL);
 				IDATA argIndexIgnoreUnrecognizedOptionsDisable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSDISABLE, NULL);
 
-				if (argIndexGcOnIdleEnable > argIndexGcOnIdleDisable) {
+				/* idle heap tuning is enabled only if -XX:+IdleTuningGcOnIdle or -Xtune:virtualized is set as VM options*/
+				if ((argIndexGcOnIdleEnable > argIndexGcOnIdleDisable) || ((argIndexGcOnIdleDisable == -1) && J9_ARE_ANY_BITS_SET(vm->runtimeFlags, J9_RUNTIME_TUNE_VIRTUALIZED))) {
```

Comment:
For readability, we typically format this larger if check like:
```
if ((argIndexGcOnIdleEnable > argIndexGcOnIdleDisable)
|| ((argIndexGcOnIdleDisable == -1) && J9_ARE_ANY_BITS_SET(vm->runtimeFlags, J9_RUNTIME_TUNE_VIRTUALIZED))
) {
```

But the check will likely change when the Java 9 only check is added.